src/Footer.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useContext } from 'react';
+import { Link } from 'react-router-dom';
+import classNames from 'classnames';
+import { Todo } from './types/Todo';
+import { FilterStatus } from './types/FilterStatus';
+import { deleteTodos } from './api/todos';
+import { TodosContext } from './TodoContext';
+
+type Props = {
+  todos: Todo[],
+  active: number,
+  completed: number,
+  filter: FilterStatus,
+  setFilter(status: FilterStatus): void,
+  setTodos(todosArray: Todo[]): void,
+};
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  active,
+  completed,
+  filter,
+  setFilter,
+  setTodos,
+}) => {
+  const { setIsDeleteError } = useContext(TodosContext);
+  const removeCompleted = async () => {
+    const completedTodos = [...todos].filter(todo => todo.completed);
+
+    setIsDeleteError(false);
```

Comment:
what about `setError(//error text)` instead of a lot of similar functions?

Reply:
and create an enum for errors

Reply:
BTW you already have comment about errors
![2023-08-08_13h31_37](https://github.com/mate-academy/react_todo-app/assets/61904995/c6d03802-55c1-4a21-9c57-6c637e627ffb)